x-pack/plugins/apm/server/routes/services/get_service_instances/get_service_instances_transaction_statistics.test.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { InstancesSortField } from '../../../../common/instances';
+import { getOrderInstructions } from './get_service_instances_transaction_statistics';
+
+describe('getOrderInstructions', () => {
+  (
+    [
+      {
+        field: 'errorRate',
+        direction: 'asc',
+        expectValue: { failures: 'asc', _count: 'desc' },
```

Comment:
Why is it called `failures` instead of `errorRate`?
```suggestion
        expectValue: { errorRate: 'asc', _count: 'desc' },
```